scripts/wrappers/plugins/remove.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+
+# Fetch the list of installed plugins
+INSTALLED_PLUGINS=$("${OPENSEARCH_HOME}"/bin/opensearch-plugin list)
+echo "${INSTALLED_PLUGINS}"
```

Comment:
this looks just like list.sh

Reply:
Good catch. I missed implementing this one